hack/build-and-push.sh

```diff
@@ -34,6 +53,10 @@ HACBS_BUNDLE_IMG=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-hacbs-templates-bundle}
 HACBS_BUNDLE_LATEST_IMG=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-hacbs-templates-bundle}:${TEST_REPO_NAME:+hacbs-}latest
 HACBS_CORE_BUNDLE_IMG=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-hacbs-core-service-templates-bundle}:${TEST_REPO_NAME:+hacbs-core-}latest
 
+OUTPUT_TASK_BUNDLE_LIST="${OUTPUT_TASK_BUNDLE_LIST-task-bundle-list}"
```

Comment:
@zregvart, taking https://github.com/hacbs-contract/ec-cli/pull/109 into account. I can change this to produce a single list of bundle images. Then, change the build-bundle task to make a single call to `ec track`. Thoughts?

Reply:
That would mean less bash, so :+1:

Reply:
Opted to keeping two lists but only calling ec-cli once with the combined list. This script produces other bundles that we do not care for (at least now). I don't want to accidentally start allowing them in the ec policy.